anonlink/entitymatch.py

```diff
@@ -33,15 +33,26 @@ def python_filter_similarity(filters1, filters2):
 
 def cffi_filter_similarity_k(filters1, filters2, k, threshold):
     """Accelerated method for determining Bloom Filter similarity.
+
+    Assumes all filters are the same length, being a multiple of 64
+    bits.
+
     """
     length_f1 = len(filters1)
     length_f2 = len(filters2)
 
-    # We assume the length is 1024 bit = 128 Bytes
-    match_one_against_many_dice_1024_k_top = lib.match_one_against_many_dice_1024_k_top
+    if length_f1 == 0:
+        return []
+
+    # Length must be a multple of 64 bits.
+    assert(len(filters1[0][0]) % 8 == 0)
```

Comment:
If you move this comment into a second argument to the assert builtin users will get a more descriptive AssertionError